### PR TITLE
cc github submission (zehui wu)

### PR DESCRIPTION
**Pull request by zehuiwu**
feature_selection_in_R.Rmd

### CODE REVIEW

File: feature_selection_in_R.Rmd [hunk 1]
```diff
@@ -0,0 +1,175 @@
...
+Zehui Wu
+
+```{r}
+library(ggcorrplot)
+library(caret)
+library(randomForest)
+library(gam)
+```
+
+## 0. Introduction
```
Comment (joywang1999) [inline feature_selection_in_R.Rmd:12]: Please remove the numbers here and all below, use only double hashtags.
```suggestion
## Introduction

...

## Correlation Matrix 

...
```

File: feature_selection_in_R.Rmd [hunk 2]
```diff
@@ -0,0 +1,175 @@
...
+```
+
+I firstly use the cor() function to calculate correlation matrix.
+
+```{r}
+cor_matrix <- data.frame(cor(winequality))
+cor_matrix
+```
+
+## 1.1 heat map for correlation matrix
```
Comment (joywang1999) [inline feature_selection_in_R.Rmd:43]: If you want to include subsection headings, use one more hashtag, and remember to remove the numbers for here and all below.
```suggestion
### heat map for correlation matrix

...

### function to find redundant variables

...
```

**Comment by joywang1999** [comment]
Please change your filename to all lowercases (ex. feature_selection_in_r.Rmd), thanks!

**Review by joywang1999 — CHANGES_REQUESTED** [review]

File: feature_selection_in_R.Rmd (hunk 2 above)
Comment (zehuiwu) [reply feature_selection_in_R.Rmd]: change has made

**Comment by joywang1999** [comment]
Hi! I saw your changed hashtags, great! But the file name is still in uppercase `feature_selection_in_R.Rmd`

File: feature_selection_in_R.Rmd (hunk 2 above)
Comment (zehuiwu) [reply feature_selection_in_R.Rmd]: Sry for being so late. File name is changed!

**Comment by joywang1999** [comment]
Thank you for your contribution! 🎉

**Pull request merged**